tests/topologyDescription/topologyDescription-getServers-001.phpt

```diff
@@ -0,0 +1,50 @@
+--TEST--
+MongoDB\Driver\TopologyDescription::getServers()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+class TopologyDescriptionProvider implements MongoDB\Driver\Monitoring\SDAMSubscriber
+{
+    private $topologyDescription;
+
+    public function topologyChanged(MongoDB\Driver\Monitoring\TopologyChangedEvent $event)
+    {
+        $this->topologyDescription = $event->getNewDescription();
+    }
+
+    public function getTopologyDescription()
+    {
+        $manager = create_test_manager();
+        $manager->addSubscriber($this);
+        $manager->executeCommand(DATABASE_NAME, new MongoDB\Driver\Command(['ping' => 1]));
+        $manager->removeSubscriber($this);
+
+        return $this->topologyDescription;
+    }
+}
+
+function assertIsServerDescriptionsArray(array $sds) {
```

Comment:
Suggested rename since this function technically isn't asserting anything. It's just returning whether the parameter is an array of SDs.

```suggestion
function isArrayOfServerDescriptions(array $sds) {
```